packages/localize/test/date/parseDate.test.js

```diff
@@ -32,8 +32,6 @@ describe('parseDate()', () => {
     expect(equalsDate(parseDate('12.12.1976'), new Date('1976/12/12'))).to.equal(true);
     expect(equalsDate(parseDate('13.12.1976'), new Date('1976/12/13'))).to.equal(true);
     expect(equalsDate(parseDate('14.12.1976'), new Date('1976/12/14'))).to.equal(true);
-    expect(equalsDate(parseDate('14.12-1976'), new Date('1976/12/14'))).to.equal(true);
```

Comment:
Could we add:

```js
expect(equalsDate(parseDate('14. 12. 1976.'), new Date('1976/12/14'))).to.equal(true);
expect(equalsDate(parseDate('14.12.1976 г.'), new Date('1976/12/14'))).to.equal(true);
```

Reply:
Added cases for above mentioned scenarios